README.md

```diff
@@ -1,100 +1,155 @@
+```
+</details>
 
-tx.PayTo("1NRoySJ9Lvby6DuE2UQYnyT67AASwNZxGb", 1000)
+<br/>
 
-wif, _ := bsvutil.DecodeWIF("KznvCNc6Yf4iztSThoMH6oHWzH9EgjfodKxmeuUGPq5DEX5maspS")
+## Examples & Tests
+All unit tests and [examples](examples) run via [Travis CI](https://travis-ci.org/libsv/go-bt) and uses [Go version 1.15.x](https://golang.org/doc/go1.15). View the [deployment configuration file](.travis.yml).
 
-signer := bt.InternalSigner{PrivateKey: wif.PrivKey, SigHashFlag: 0}
-err := tx.SignAuto(&signer)
-if err != nil {
-	fmt.Errorf(err.Error())
-}
+Run all tests (including integration tests)
+```shell script
+make test
+```
 
-fmt.Println(tx.ToString())
+Run tests (excluding integration tests)
+```shell script
+make test-short
 ```
 
-prints:
+<br/>
 
-```console
-01000000019d81a1815050f3a7bf8392e077481151af50a011a10708d4fc480a8ead76b411000000006b483045022100dda18196d5217ecfe01390a7ec9c0bd577e7d97ed88f92b7c4a2bf8cb94a493b0220465f9ab035ae584d45c0fbb41363c1cd862b8439619b3b42decb1e9f556dd142412102798913bc057b344de675dac34faafe3dc2f312c758cd9068209f810877306d66ffffffff01e8030000000000001976a914eb0bd5edba389198e73f8efabddfc61666969ff788ac00000000
+## Benchmarks
+Run the Go [benchmarks](tx_test.go):
+```shell script
+make bench
 ```
 
-#### Regular P2PKH + OP_RETURN output
-
-```go
-	tx := bt.NewTx()
+<br/>
 
-	err := tx.From(
-		"b7b0650a7c3a1bd4716369783876348b59f5404784970192cec1996e86950576",
-		0,
-		"76a9149cbe9f5e72fa286ac8a38052d1d5337aa363ea7f88ac",
-		1000)
+## Code Standards
+Read more about this Go project's [code standards](CODE_STANDARDS.md).
 
-	tx.PayTo("1C8bzHM8XFBHZ2ZZVvFy2NSoAZbwCXAicL", 900)
+<br/>
 
-	o, err := bt.NewOpReturnOutput([]byte("You are using LiBSV!"))
-	if err != nil {
-		fmt.Println(err.Error())
-	}
+## Usage
+View the [examples](examples)
 
-	tx.AddOutput(o)
+<br/>
 
-	wif, _ := bsvutil.DecodeWIF("L3VJH2hcRGYYG6YrbWGmsxQC1zyYixA82YjgEyrEUWDs4ALgk8Vu")
+## Maintainers
+| [<img src="https://github.com/jadwahab.png" height="50" alt="JW" />](https://github.com/jadwahab) | [<img src="https://github.com/mrz1836.png" height="50" alt="MrZ" />](https://github.com/mrz1836) |
+|:---:|:---:|
+| [JW](https://github.com/jadwahab) | [MrZ](https://github.com/mrz1836) |
 
-	signer := bt.InternalSigner{PrivateKey: wif.PrivKey, SigHashFlag: 0}
-	err = tx.SignAuto(&signer)
-	if err != nil {
-		fmt.Println(err.Error())
-	}
+<br/>
 
-	fmt.Println(tx.ToString())
-```
+## Contributing
+View the [contributing guidelines](CONTRIBUTING.md) and please follow the [code of conduct](CODE_OF_CONDUCT.md).
 
-prints:
+### How can I help?
+All kinds of contributions are welcome :raised_hands:!
+The most basic way to show your support is to star :star2: the project, or to raise issues :speech_balloon:.
+You can also support this project by [becoming a sponsor on GitHub](https://github.com/sponsors/libsv) :clap:
+or by making a [**bitcoin donation**](https://gobitcoinsv.com/#sponsor) to ensure this journey continues indefinitely! :rocket:
```

Comment:
Same link here, for where to donate lol

Reply:
i'm fine to leave it on gobitcoinsv.com for now, enjoy haha